bridge-impl/src/main/java/com/liferay/faces/bridge/context/url/internal/BridgeURLBase.java

```diff
@@ -444,9 +444,16 @@ protected boolean matchPathAndExtension(String file1, String file2) {
 					String[] queryParameters = queryString.split("[&]");
 
 					for (String queryParameter : queryParameters) {
+
 						String[] nameValueArray = queryParameter.split("[=]");
 
-						if (nameValueArray.length == 2) {
+						if (nameValueArray.length == 1) {
+
+							String name = nameValueArray[0];
+							requestParameters.add(new RequestParameter(name, ""));
+						}
+						else if (nameValueArray.length == 2) {
+
 							String name = nameValueArray[0];
 							String value = nameValueArray[1];
 							requestParameters.add(new RequestParameter(name, value));
```

Comment:
@ngriffin7a,
**Side note:** Should we thrown a `MalformedURLException` when `"".equals(name)`?